### PR TITLE
(dir/sw) - correct exports

### PR DESCRIPTION
**Pull request by JoviDeCroock**

### CODE REVIEW

File: packages/directory-plugin/package.json [hunk 1]
```diff
@@ -2,6 +2,12 @@
 	"name": "@wmr-plugins/directory-import",
 	"version": "0.1.0",
 	"type": "module",
+	"main": "./directory-plugin.cjs",
+	"module": "./src/index.js",
+	"exports": {
+		"import": "./src/index.js",
+		"require": "./directory-plugin.cjs"
```
Comment (developit) [inline packages/directory-plugin/package.json:9]:
```suggestion
		"default": "./directory-plugin.cjs"
```

File: packages/sw-plugin/package.json [hunk 2]
```diff
@@ -2,6 +2,12 @@
 	"name": "@wmr-plugins/service-worker",
 	"version": "0.1.0",
 	"type": "module",
+	"main": "./sw-plugin.cjs",
+	"module": "./src/index.js",
+	"exports": {
+		"import": "./src/index.js",
+		"require": "./sw-plugin.cjs"
```
Comment (developit) [inline packages/sw-plugin/package.json:9]:
```suggestion
		"default": "./sw-plugin.cjs"
```

**Review by developit — APPROVED** [review]
lgtm with the default swap

**Comment by developit** [comment]
@JoviDeCroock QQ: for the SW plugin, I'm thinking the rollup'd `.cjs` dist file contains a copy of Rollup, whereas `src/index.js` just depends on it?

**Comment by JoviDeCroock** [comment]
@developit it doesn't embed a copy as it's an async import for dev-only

**Pull request merged**